### ISSUE TITLE
Qlik Object 

### ISSUE THREAD

**Issue by burrav93** [motor-js/motor-ui#13]
We connected to the Engine, was able to use hypercube to get data and pass it into the charts. But, when I use the "Qlik Object" and give an id , it just spins but doesn't load any object.  I'm trying to get a filter from my app but it just does not display that object . I don't see any error though. Not just filters, but I have tried KPI , Line charts etc. But , I just don't see any object rendering . Any help is appreciated! 

Thanks,
karthik

**Comment by burrav93** [comment]
The Qlik object uses "Capability Context" instead of "Engine Context" . 
We used this capability context in my app and console log the app to basically check if the app is connected. It says undefined. Is there any other lines of code that we need to add or change?

**Comment by lureilly1** [comment]
Hey Karthik, in order to use the QlikObject, you need to set the capabilityAPI prop to true in the Motor component. Have you already done this?

**Comment by burrav93** [comment]
Hi Luke,

Thanks. We added the prop capabilityAPI to true. Also had to do few more things - adding prefix as empty string in the config and then we got another issue to avoid CORS blocking. We  set the virtual proxy with the setting Access control allow origin in QMC and we fixed the error. Then we were able to get the qlik objects from Qlik app. 

Now the objects loads, but we see this error "TypeError: Cannot read property 'resize' of null" . This error pops when we resize the screen. The error points to line 71 in QlikObject.jsx . It has to do something with qViz.resize() . Any help is appreciated. Thank you
<img width="473" alt="resizeerror" src="https://user-images.githubusercontent.com/31553172/111494892-d8b48a80-870c-11eb-9aa6-a6e1bcec406e.PNG">
